### ISSUE TITLE
Can not install msodbcsql17 package distribution

### ISSUE THREAD

**Issue by akihiro-inui** [dotnet/core#6386]
### Description
Hi, I am trying to download the driver for MSSQL database 17 on Ubuntu 18.04.
The following command used to download the driver but it can not find the package anymore and I am getting 404 status code.

```
 sudo apt-get install -y msodbcsql17
```
and I am getting

```
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  msodbcsql17
0 upgraded, 1 newly installed, 0 to remove and 5 not upgraded.
Need to get 745 kB of archives.
After this operation, 0 B of additional disk space will be used.
Err:1 https://packages.microsoft.com/ubuntu/18.04/prod bionic/main amd64 msodbcsql17 amd64 17.7.2.1-1
  404  Not Found [IP: 13.81.215.193 443]
E: Failed to fetch https://packages.microsoft.com/ubuntu/18.04/prod/pool/main/m/msodbcsql17/msodbcsql17_17.7.2.1-1_amd64.deb  404  Not Found [IP: 13.81.215.193 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

### Configuration
OS: Ubuntu 18.04

**Comment by jlyshoel** [comment]
Also experiencing the same problem here. We're running debian stretch in a docker container. Starting from a clean init of the container, we had it working yesterday.

Same error 

```
2021-06-17T06:39:13.8172870Z [0mReading package lists...
2021-06-17T06:39:13.9776768Z Building dependency tree...
2021-06-17T06:39:13.9781330Z Reading state information...
2021-06-17T06:39:14.0868387Z The following additional packages will be installed:
2021-06-17T06:39:14.0873957Z   libodbc1 multiarch-support odbcinst odbcinst1debian2 unixodbc unixodbc-dev
2021-06-17T06:39:14.0878105Z Suggested packages:
2021-06-17T06:39:14.0880367Z   unixodbc-bin
2021-06-17T06:39:14.1166609Z The following NEW packages will be installed:
2021-06-17T06:39:14.1171116Z   msodbcsql17 multiarch-support unixodbc
2021-06-17T06:39:14.1175190Z The following packages will be upgraded:
2021-06-17T06:39:14.1179265Z   libodbc1 odbcinst odbcinst1debian2 unixodbc-dev
2021-06-17T06:39:14.1541075Z 4 upgraded, 3 newly installed, 0 to remove and 0 not upgraded.
2021-06-17T06:39:14.1541589Z Need to get 1721 kB of archives.
2021-06-17T06:39:14.1541976Z After this operation, 253 kB of additional disk space will be used.
2021-06-17T06:39:14.1542410Z Err:1 https://packages.microsoft.com/debian/10/prod buster/main amd64 odbcinst amd64 2.3.7
2021-06-17T06:39:14.1542813Z   404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0305852Z Err:2 https://packages.microsoft.com/debian/10/prod buster/main amd64 unixodbc-dev amd64 2.3.7
2021-06-17T06:39:15.0308680Z   404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0309376Z Err:3 https://packages.microsoft.com/debian/10/prod buster/main amd64 odbcinst1debian2 amd64 2.3.7
2021-06-17T06:39:15.0310007Z   404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0310698Z Err:4 https://packages.microsoft.com/debian/10/prod buster/main amd64 libodbc1 amd64 2.3.7
2021-06-17T06:39:15.0311181Z   404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0311594Z Err:5 https://packages.microsoft.com/debian/10/prod buster/main amd64 unixodbc amd64 2.3.7
2021-06-17T06:39:15.0311995Z   404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0312929Z Err:6 https://packages.microsoft.com/debian/10/prod buster/main amd64 msodbcsql17 amd64 17.7.2.1-1
2021-06-17T06:39:15.0313350Z   404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0313951Z Get:7 http://deb.debian.org/debian buster/main amd64 multiarch-support amd64 2.28-10 [215 kB]
2021-06-17T06:39:15.0314792Z E: Failed to fetch https://packages.microsoft.com/debian/10/prod/pool/main/u/unixodbc/odbcinst_2.3.7_amd64.deb  404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0315732Z E: Failed to fetch https://packages.microsoft.com/debian/10/prod/pool/main/u/unixodbc/unixodbc-dev_2.3.7_amd64.deb  404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0316646Z E: [0mFetched 215 kB in 0s (3522 kB/s)
2021-06-17T06:39:15.0317402Z Failed to fetch https://packages.microsoft.com/debian/10/prod/pool/main/u/unixodbc/odbcinst1debian2_2.3.7_amd64.deb  404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0318361Z E: Failed to fetch https://packages.microsoft.com/debian/10/prod/pool/main/u/unixodbc/libodbc1_2.3.7_amd64.deb  404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0319054Z E: Failed to fetch https://packages.microsoft.com/debian/10/prod/pool/main/u/unixodbc/unixodbc_2.3.7_amd64.deb  404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0320022Z E: Failed to fetch https://packages.microsoft.com/debian/10/prod/pool/main/m/msodbcsql17/msodbcsql17_17.7.2.1-1_amd64.deb  404  Not Found [IP: 137.135.134.16 443]
2021-06-17T06:39:15.0320875Z E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

**Comment by attrib** [comment]
Seems like the complete pool directory with all deb packages is gone? I can't find any deb package anymore on packages.microsoft.com. So not a problem with the index file, all packages are gone.

**Comment by jlyshoel** [comment]
> > Seems like the complete pool directory with all deb packages is gone? I can't find any deb package anymore on packages.microsoft.com. So not a problem with the index file, all packages are gone.
> 
> Yes, the pool directory is gone. Not sure if this issue should be addressed here or elsewhere. Let me know if there is a better place to report this issue.

These might be related or the same issue: https://github.com/dotnet/core/issues/6383 https://github.com/dotnet/core/issues/6381

**Comment by Lucario95** [comment]
I have the same issue

**Comment by vkuehn** [comment]
And related to https://github.com/dotnet/core/issues/4122

**Comment by HeavyKid** [comment]
Same issue here.

**Comment by Lucario95** [comment]
This should now be resolved as stated by #6381

**Comment by michaelgsharp** [comment]
This issue is being tracked with #6381. It should be fixed now (as Lucario95 said), but some are still seeing slowdowns since everyone is trying to download it. Closing this issue.
